src/Corax/Constants.cs

```diff
@@ -7,7 +7,12 @@ namespace Corax
     public static class Constants
     {
         public const string NullValue = "NULL_VALUE";
+        public static ReadOnlySpan<char> NullValueSpan => NullValue;
```

Comment:
I think that the usage of `=>` will cause the conversion every time you access `NullValueSpan`. Let's calculate it just once.

Reply:
Done, in the case of byte array this pattern is no-allocating, but unfortunately, it's only for bytes.

Reply:
That's correct, bytes only. You could though do that internally with bytes and the access is a casting to char. I do that for some tables in SIMD algorithms.